src/main/scala/com/mwz/sonar/scala/sensor/Measures.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Sonar Scala Plugin
+ * Copyright (C) 2018 All contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package com.mwz.sonar.scala.sensor
+
+import scala.annotation.tailrec
+import scalariform.lexer.{Token, Tokens}
+
+/** Scala Sensor Metrics */
+object Measures {
+  def countClasses(tokens: List[Token]): Int = {
+    tokens.foldLeft(0) {
+      case (acc, token) =>
+        val tokenType = token.tokenType
+        if (tokenType == Tokens.CLASS || tokenType == Tokens.OBJECT) acc + 1
+        else acc
+    }
+  }
+
+  def countMethods(tokens: List[Token]): Int = {
+    tokens.foldLeft(0) {
+      case (acc, token) =>
+        if (token.tokenType == Tokens.DEF) acc + 1
+        else acc
+    }
+  }
+
+  /* applied on raw source code */
+
+  /* applied on lines of code */
+
+  /* applied on tokenized code */
```

Comment:
What are the above comments referring to?

Reply:
No idea, not mine.
I only change the package name of that file.

Reply:
Should we get rid of those then?

Reply:
I think yes, we should.
I also found them confusing.